javascript/javascript1/week3/week3.js

```diff
@@ -0,0 +1,135 @@
+//username on freecodecamp: Rikkaa
+//Item array removal
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+
+// Write some code here
+
+const shiftName = names.shift(function (e) {
+  return e !== nameToRemove;
+});
+// Code done
+console.log(names); // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']
+
+//When will we be there??
+
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 432,
+};
+
+function timefunction(destinationDistance, speed) {
+  return travelInformation.destinationDistance / travelInformation.speed;
+}
+
+const travelTime = timefunction(travelInformation);
+console.log(
+  Math.floor(travelTime) +
+    " " +
+    "hours and " +
+    Math.floor((travelTime - Math.floor(travelTime)) * 60) +
+    " " +
+    "mintues"
+); // 8 hours and 38 minutes
+
+//Series duration of my life
+const seriesDurations = [
+  {
+    title: "The wheel of time",
+    days: 0,
+    hours: 7,
+    minutes: 28,
+  },
+  {
+    title: "Rita",
+    days: 1,
+    hours: 1,
+    minutes: 20,
+  },
+  {
+    title: "Sex education",
+    days: 1,
+    hours: 2,
+    minutes: 40,
+  },
+];
+const lifeTime = 80 * 365 * 24 * 60;
+
+function seriesTime() {
+  for (let i = 0; i < seriesDurations.length; i++) {
+    let logDuration =
+      ((seriesDurations[i].days * 24 * 60 +
+        seriesDurations[i].hours * 60 +
+        seriesDurations[i].minutes) /
+        lifeTime) *
+      100;
+    console.log(
+      seriesDurations[i].title +
+        " took " +
+        logDuration.toFixed(seriesDurations.length) +
+        "% of my life"
+    );
+  }
+}
+seriesTime(seriesDurations);
```

Comment:
Your function doesn't take any argument, so you can either call it without it or remake the seriesTime function to takes one 🤠